packages/util/src/twox64Concat.ts

```diff
@@ -4,5 +4,6 @@ import { xxhashAsU8a } from "@polkadot/util-crypto"
 const bitLength = 64
 
 export function twox64Concat(input: string | Buffer | Uint8Array): `0x${string}` {
-  return u8aToHex(u8aConcat(xxhashAsU8a(input, bitLength), u8aToU8a(input)))
+  const inputAsU8a = typeof input === "string" ? input : new Uint8Array(input)
```

Comment:
I'm not sure about this - I suspect when `input` is a `string`, it might be a hex string?
In which case, I don't think `new Uint8Array(hexString)` correctly decodes a hex string into a Uint8Array.